packages/components/src/Menu/Menu.css

```diff
@@ -8,27 +8,53 @@
 }
 
 .menu {
-  position: absolute;
+  position: fixed;
   right: 0;
+  bottom: 0;
+  left: 0;
   z-index: var(--elevation-menu);
-  width: calc(var(--base-unit) * 12.5);
-  max-height: 300px;
-  box-shadow: var(--shadow-base);
+  width: 100%;
+  max-height: 72vh;
   padding: var(--space-small);
+  padding-bottom: env(safe-area-inset-bottom);
   border: var(--border-base) solid var(--color-grey--lighter);
-  border-radius: var(--radius-base);
-  overflow: auto;
+  border-radius: var(--radius-base) var(--radius-base) 0 0;
+  overflow-y: scroll;
   background-color: var(--color-white);
+  -webkit-overflow-scrolling: touch;
 }
 
-.above {
-  bottom: 100%;
-  margin-bottom: var(--space-smallest);
+@media screen and (max-width: 639px), print {
+  .menu:last-child {
+    padding-bottom: var(--space-small);
+  }
 }
 
-.below {
-  top: 100%;
-  margin-top: var(--space-smallest);
+@media only screen and (min-width: 640px), print {
+  .menu {
+    position: absolute;
+    left: auto;
+    z-index: var(--elevation-menu);
+    width: calc(var(--base-unit) * 12.5);
+    max-height: 300px;
```

Comment:
should we use a `calc(var(--base-unit) * 18.75)` for this number here?

Reply:
Or, even a rounder number would be nice. I wonder if we could *20 it? @chris-at-jobber

Reply:
On second look at the dropdowns in Jobber right now we shouldn't even be setting a max-height on this unless we have a good plan to revisit the entire menu pattern

Reply:
I wonder if max hight should be more something like `90vh` so it scrolls but only if it's REALLY BIG?

Reply:
Probably the nicest way to solve for this is "let the menu grow from (the nearest point to the button) to (16px from the top or bottom of the viewport) and then scroll if there's overflow", which I think I could hack out in jQuery 😆 ...

Reply:
Oh, I was thinking of the responsive version with the above. I agree with you for the non-responsive tough.

Reply:
Yeah max-height of 72vh on the small one is intentional to ensure there's always adequate tap-to-close room at the top of the view

Reply:
(Client More Actions is probably the best example) 
![image](https://user-images.githubusercontent.com/39704901/85075841-4cc10280-b17c-11ea-9179-f4fab98a6956.png)